src/views/forms/FormProfileCreation/FormProfileCreationTs.ts

```diff
@@ -150,13 +210,33 @@ export class FormProfileCreationTs extends Vue {
         // use repository for storage
         this.accountService.saveProfile(account);
 
-        // execute store actions
-        this.$store.dispatch('profile/SET_CURRENT_PROFILE', account);
-        this.$store.dispatch('temporary/SET_PASSWORD', this.formItems.password);
-        this.$store.dispatch('notification/ADD_SUCCESS', NotificationType.OPERATION_SUCCESS);
+        if (!this.isLedger) {
+            // execute store actions
+            this.$store.dispatch('profile/SET_CURRENT_PROFILE', account);
+            this.$store.dispatch('temporary/SET_PASSWORD', this.formItems.password);
+            this.$store.dispatch('notification/ADD_SUCCESS', NotificationType.OPERATION_SUCCESS);
 
-        // flush and continue
-        this.$router.push({ name: this.nextPage });
+            // flush and continue
+            this.$router.push({ name: this.nextPage });
+        } else {
+            // execute store actions
+            this.$store.dispatch('profile/SET_CURRENT_PROFILE', account);
+            this.$store.dispatch('temporary/SET_PASSWORD', this.formItems.password);
```

Comment:
These two lines for executing store actions are redundant, take both of them out of conditions and use them once.

Reply:
Okay, i will update it